searcch_backend/api/app.py

```diff
@@ -133,4 +133,6 @@ def log_request_info():
 api.add_resource(BadgeResource, approot + '/badge/<int:badge_id>', endpoint='api.badge')
 
 api.add_resource(LicenseResourceRoot, approot + '/licenses', endpoint='api.licenses')
-api.add_resource(LicenseResource, approot + '/license/<int:org_id>', endpoint='api.license')
\ No newline at end of file
+api.add_resource(LicenseResource, approot + '/license/<int:org_id>', endpoint='api.license')
+
+api.add_resource(AdminUpdatePrivileges, approot + '/admin/update_privileges', endpoint='api.admin')
```

Comment:
I would get rid of the `Update` and `update_` tokens on these endpoints... just use the noun `privileges`.  The HTTP methods are the verbs; in a RESTful API, they (almost) always address (operate on) nouns.

Reply:
Also, I would use a different noun -- `user`.  Privilege is just one aspect of the user.  Down the road they might ask us to change their email addr as well, and many other things.